src/gt4py/backend/gt_backends.py

```diff
@@ -163,6 +163,7 @@ class GTPyExtGenerator(gt_ir.IRNodeVisitor):
         gt_ir.BinaryOperator.MUL: "*",
         gt_ir.BinaryOperator.DIV: "/",
         gt_ir.BinaryOperator.POW: lambda lhs, rhs: "pow({lhs}, {rhs})".format(lhs=lhs, rhs=rhs),
+        gt_ir.BinaryOperator.MOD: lambda lhs, rhs: "fmod({lhs}, {rhs})".format(lhs=lhs, rhs=rhs),
```

Comment:
```suggestion
        gt_ir.BinaryOperator.MOD: lambda lhs, rhs: f"fmod({lhs}, {rhs})",
```

If you accept this optional suggestion, make sure that `POW` is also changed accordingly

Reply:
Further shortened to `"fmod({}, {})".format`.